src/mongocxx/test/spec/change_stream.cpp

```diff
@@ -41,79 +35,96 @@ using namespace bsoncxx;
 using namespace bsoncxx::string;
 using namespace spec;
 using namespace test_util;
+using bsoncxx::builder::basic::kvp;
+using bsoncxx::builder::basic::make_array;
+using bsoncxx::builder::basic::make_document;
+
+bool should_skip(const array::element& test) {
+    if (test["description"].get_utf8().value.compare(
+            "Change Stream should error when an invalid aggregation stage is passed in") == 0) {
+        UNSCOPED_INFO(
+            "Skipping test with invalid pipeline stages. The C++ driver cannot test them.");
+        return true;
+    }
+    return should_skip_spec_test({uri{}}, test.get_document().value);
+}
+
+void test_setup(document::view test, document::view test_spec) {
```

Comment:
Just pulled this logic out like in `transactions.cpp` and outsourced some of the work to the function in `util.cpp`